exampleSite/content/shortcodes/openapi/_index.en.md

```diff
@@ -1,56 +0,0 @@
----
```

Comment:
Should this be deleted?

Reply:
Accidentally removed the whole file instead of my changes, its back now